tardis/io/model/readers/util.py

```diff
@@ -39,16 +39,15 @@ def read_csv_isotope_abundances(
     Returns
     -------
     index : np.ndarray
-    abundances : pandas.DataFrame
-    isotope_abundance : pandas.MultiIndex
+    mass_fractions : pandas.DataFrame
+    isotope_mass_fraction : pandas.MultiIndex
     """
-
     df = pd.read_csv(
         fname, comment="#", sep=delimiter, skiprows=skip_rows, index_col=0
     )
     df = df.transpose()
 
-    abundance = pd.DataFrame(
+    mass_fraction = pd.DataFrame(
```

Comment:
I would think this should be plural here, like line 42 suggests. "mass fraction" sounds like a different thing to me. This also applies to line 59, isotope_mass_fraction -> isotope_mass_fractions